README.md

```diff
@@ -56,7 +56,19 @@ list of prefixes:
 
     godepgraph -p github.com,launchpad.net bitbucket.org/foo/bar
 
-
+### As part of larger project (e.g. Docker):
+
+```
+mkdir -p go2/src/github.com/docker
```

Comment:
The thing is that I need to emphasize the need to run it within the directory which isn't intuitive because the code take it from os.Getwd